airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -200,21 +200,31 @@ export const DagsList = () => {
           <Heading py={3} size="md">
             {pluralize("DAG", data?.total_entries)}
           </Heading>
-          {display === "card" ? (
```

Comment:
Let's keep the select only for the card list. But let's drop "sort by last run state", I think last run start_date is fine.

Reply:
We can pass a custom `sortingFn` to sort the table view by start_date to the latest_dag_runs column definition.

Reply:
Got it.